modules/hcp-dc-image-mirror.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy-disconnected.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-dc-image-mirror_{context}"]
+= Configuring image mirroring for {hcp} in a disconnected environment
+
+Image mirroring is the process of fetching images from external registries, such as `registry.redhat.com` or `quay.io`, and storing them in your private registry.
+
+In the following procedures, the `oc-mirror` tool is used, which is a binary that uses the `ImageSetConfiguration` object. In the file, you can specify the following information:
+
+* The {product-title} versions to mirror. The versions are in `quay.io`.
+* The additional Operators to mirror. Select packages individually.
+* The extra images that you want to add to the repository.
+
+.Prerequisites
+
+Ensure that the registry server is running before you start the mirroring process.
+
+.Procedure
+
+To configure image mirroring, complete the following steps:
+
+. Ensure that your `${HOME}/.docker/config.json` file is updated with the registries that you are going to mirror from and with the private registry that you plan to push the images to.
+
+. By using the following example, create an `ImageSetConfiguration` object to use for mirroring. Replace values as needed to match your environment:
++
+[source,yaml]
+----
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+storageConfig:
+  registry:
+    imageURL: registry.<dns.base.domain.name>:5000/openshift/release/metadata:latest <1>
+mirror:
+  platform:
+    channels:
+    - name: candidate-4.14
+      minVersion: 4.x.y-x86_64  <2>
+      maxVersion: 4.x.y-x86_64 <2>
+      type: ocp
+    graph: true
+  additionalImages:
+  - name: quay.io/karmab/origin-keepalived-ipfailover:latest
+  - name: quay.io/karmab/kubectl:latest
+  - name: quay.io/karmab/haproxy:latest
+  - name: quay.io/karmab/mdns-publisher:latest
+  - name: quay.io/karmab/origin-coredns:latest
+  - name: quay.io/karmab/curl:latest
+  - name: quay.io/karmab/kcli:latest
+  - name: quay.io/user-name/trbsht:latest
+  - name: quay.io/user-name/hypershift:BMSelfManage-v4.14-rc-v3
+  - name: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.10
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.14
+    packages:
+    - name: lvms-operator
+    - name: local-storage-operator
+    - name: odf-csi-addons-operator
+    - name: odf-operator
+    - name: mcg-operator
+    - name: ocs-operator
+    - name: metallb-operator
+    - name: kubevirt-hyperconverged <3>
+----
++
+<1> Replace `<dns.base.domain.name>` with the DNS base domain name.
+<2> Replace `4.x.y` with the supported {product-title} version you want to use.
+<3> For deployments on the IPv4 network, include this line.
+
+. Start the mirroring process by entering the following command:
++
+----
+$ oc-mirror --source-skip-tls --config imagesetconfig.yaml docker://${REGISTRY}
```

Comment:
Why do we need `--source-skip-tls`?
We also need to pass `--v2` when using oc-mirror v2 with `kubeVirtContainer: true` flag in the `ImageSetConfiguration` (`mirror.openshift.io/v2alpha1` only) to mirror the RHCOS boot image for the Kubevirt provider.

Reply:
@jparrill or @davidvossel Can either of you answer @tiraboschi's question about `--source-skip-tls`? Does that flag need to be included?

Reply:
In order to use `apiVersion: mirror.openshift.io/v1alpha2` for `kubeVirtContainer: true`, we also need --v2 here.